README.md

```diff
@@ -14,6 +17,30 @@ JavaScript frameworks.
 It's designed to work with Rails 5.1+ and makes use of the [Yarn](https://yarnpkg.com) dependency management
 that's been made default from that version forward.
 
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Binstubs](#binstubs)
+- [Configuration](#configuration)
+- [Advanced Configuration](#advanced-configuration)
+- [Linking to static assets](#linking-to-static-assets)
+- [Getting asset path](#getting-asset-path)
+- [Deployment](#deployment)
+- [Linking to sprockets assets](#linking-to-sprockets-assets)
+- [Ready for React](#ready-for-react)
```

Comment:
:bulb: BTW, if you want the Ruby equivalent of DocToc, you might want to check out [Tocer](https://github.com/bkuhlmann/tocer). I use it for all my projects (both for private and public repos). The nice thing about Tocer is that you can add it as a gem dependency to the project and wire it up a check for README changes prior to deploy (or even part of the build process).